benches/benches/grisubal/grid_size.rs

```diff
@@ -0,0 +1,47 @@
+// ------ IMPORTS
+
+use criterion::{black_box, criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+use honeycomb::prelude::{
+    grisubal::{grisubal, Clip},
+    CMap2,
+};
+use honeycomb_benches::FloatType;
+
+// ------ CONTENT
+
+pub fn criterion_benchmark(c: &mut Criterion) {
+    // passing args to cargo bench filters bench instead of actually reading args;
+    // the path to the file needs to be hardcoded (I think?)
+    // let args: Vec<String> = std::env::args().collect();
+    let path = "path/to/file.vtk";
```

Comment:
How does it work?

Reply:
I added a comment to the line. Basically, this str should be replaced with a path to an actual input by whoever runs the benchmark. 

An alternative would be to add whatever input we want to benchmark in the repo, so that we can add the correct path.

Reply:
Put at least a file in the repo to make sure everything is working without having to modify the files.

Reply:
it's done!